### DISCUSSION TITLE
Publishing on OpenVSX

### DISCUSSION THREAD

**Discussion by PeterWone** [PDConSec/vsc-print#222] [Polls]
OpenVSX is another marketplace for extensions than isn't run by Microsoft. Depending on the level of interest we may reopen #120.

**Comment by ghost** [comment]
Would you be able to publish to OpenVSX now that most respondents want to use it?
Thank you for the great extension!

**Reply by PeterWone** [reply]
Guess so. There are a few things that must happen first though. 

- I'll have to jump through their hoops to prove my identity.
- We're having a few issues with the automation for localisation to multiple languages.
- Got a couple of bugs in the backlog.

**Reply by ghost** [reply]
Thank you, much appreciated!

**Comment by timnew** [comment]
@PeterWone any update on this issue? Any plan to bring it back to openVSX?
Trying to print markdown from Cursor, and cannot find this extension, and realise it was taken down.

**Comment by ggedde** [comment]
Same here, using Cursor
